js/modules/k6/http/http.go

```diff
@@ -71,12 +79,18 @@ func (r *RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
 	mustExport("get", func(url sobek.Value, args ...sobek.Value) (*Response, error) {
 		// http.get(url, params) doesn't have a body argument, so we add undefined
 		// as the third argument to http.request(method, url, body, params)
+
+		// get method should not have more than two arguments
+		validateArgCount("get", args...)
 		args = append([]sobek.Value{sobek.Undefined()}, args...)
 		return mi.defaultClient.Request(http.MethodGet, url, args...)
```

Comment:
```suggestion
		// get method should not have more than two arguments
		validateArgCount("get", args...)
		// http.get(url, params) doesn't have a body argument, so we add undefined
		// as the third argument to http.request(method, url, body, params)
		args = append([]sobek.Value{sobek.Undefined()}, args...)
		return mi.defaultClient.Request(http.MethodGet, url, args...)
```

Nit; I'd prefer if we can keep the order as suggested here, as I think the already existing comment is tied to the `args` slice we build below, so it kinda makes more sense to keep both together. Same for the `"head"` method.

Could you confirm it, please @olegbespalov?